app-web/registry/information-and-application-security.json

```diff
@@ -46,6 +57,20 @@
           "description": "Application Security in a Cloud Native World.",
           "url": "https://raw.githubusercontent.com/bcgov/devhub-resources/master/resources/security/BCGov-DevSecOps-Project.pdf"
         }
+      },
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/BCDevOps/platform-services/tree/master/security/aporeto/docs",
+          "owner": "BCDevOps",
+          "repo": "platform-services",
+          "files": [
+            "README.md",
```

Comment:
are these files relative to `docs`?

the paths absolute from the root directory of the repo.

Reply:
➜  docs git:(feature/devdocs) pwd
/Users/jleach/Workspace/bcdevx/platform-services/security/aporeto/docs
➜  docs git:(feature/devdocs) tree .
.
├── CustomPolicy.md
├── ExternalNetworks.md
├── QuickStart.md
├── README.md
├── add_labels.gif
├── add_nsp.gif
├── sample
│   └── quickstart-nsp.yaml
└── webview_nsp.gif

1 directory, 8 files
➜  docs git:(feature/devdocs)

Reply:
LMK if I need to change something.

Reply:
yep should be based off the root directory

`security/aporeto/docs/CustomPolicy.md`

Reply:
Fixed.